Composer/packages/client/src/components/CreationFlow/v2/AddBotModal.tsx

```diff
@@ -50,7 +50,7 @@ export const AddBotModal: React.FC<AddBotModalProps> = (props) => {
       title={formatMessage('Add a bot')}
       onDismiss={props.onDismiss}
     >
-      <ChoiceGroup required defaultSelectedKey={addExistingKey} options={addSkillOptions} onChange={handleChange} />
+      <ChoiceGroup required defaultSelectedKey={addNewBotKey} options={addSkillOptions} onChange={handleChange} />
```

Comment:
Is there a reason we're changing this default?

Reply:
Yep, Capn' Scott and design requested it